libs/feature/editor/src/lib/components/record-form/form-field/form-field-rich/form-field-rich.component.html

```diff
@@ -12,10 +12,10 @@
     <span
       class="material-symbols-outlined gn-ui-icon-small"
       [ngClass]="{ 'text-primary': !preview }"
-      >{{ preview ? 'visibility' : 'visibility_off' }}</span
+      >{{ preview ? 'visibility_off' : 'visibility' }}</span
     >
     &nbsp;
-    {{ preview ? 'WYSIWYG' : 'Markdown' }}
+    {{ preview ? 'Edit' : 'Preview' }}
```

Comment:
that looks inverted as well? I thought `preview` meant the text was rendered properly without being editable

Reply:
Same here, it was intentional. The button shows what you can do (action), not the current state.